assemblies/upgrading-odh-v1.adoc

```diff
@@ -0,0 +1,33 @@
+:_module-type: ASSEMBLY
+
+ifdef::context[:parent-context: {context}]
+
+:context: upgradev1
+
+[id="upgrading-odh-v1_{context}"]
+= Upgrading Open Data Hub version 1
+
+[role='_abstract']
+You can upgrade the Open Data Hub Operator from version 1 to version 2 using the OpenShift web console. 
```

Comment:
*by* using the OpenShift console. (don't need "web")
add "by using" to help non-native English speakers interpretation of the sentence